src/vs/workbench/contrib/welcome/telemetryOptOut/browser/telemetryOptOut.ts

```diff
@@ -62,8 +62,8 @@ export abstract class AbstractTelemetryOptOut implements IWorkbenchContribution
 	}
 
 	private showTelemetryOptOut(telemetryOptOutUrl: string): void {
-		const optOutNotice = localize('telemetryOptOut.optOutNotice', "Help improve VS Code by allowing Microsoft to collect usage data. Read our [privacy statement]({0}) and learn how to [opt out]({1}).", this.privacyUrl, this.productService.telemetryOptOutUrl);
-		const optInNotice = localize('telemetryOptOut.optInNotice', "Help improve VS Code by allowing Microsoft to collect usage data. Read our [privacy statement]({0}) and learn how to [opt in]({1}).", this.privacyUrl, this.productService.telemetryOptOutUrl);
+		const optOutNotice = localize('telemetryOptOut.optOutNotice', "Help improve Azure Data Studio by allowing Microsoft to collect usage data. Read our [privacy statement]({0}) and learn how to [opt out]({1}).", this.privacyUrl, this.productService.telemetryOptOutUrl);
```

Comment:
Probably adding a SQL CARBON EDIT tag would be a good idea to avoid this in future.